website/source/docs/internals/anti-entropy.html.markdown

```diff
@@ -0,0 +1,131 @@
+---
+layout: "docs"
+page_title: "Anti-Entropy"
+sidebar_current: "docs-internals-anti-entropy"
+description: >
+  This section details the process and use of anti-entropy in Consul.
+---
+
+# Anti-Entropy
+
+Consul uses an advanced method of maintaining service and health information.
+This page details how services and checks are registered, how the catalog is
+populated, and how health status information is updated as it changes.
+
+~> **Advanced Topic!** This page covers technical details of
+the internals of Consul. You don't need to know these details to effectively
+operate and use Consul. These details are documented here for those who wish
+to learn about them without having to go spelunking through the source code.
+
+### Components
+
+It is important to first understand the moving pieces involved in services and
+health checks: the [agent](#agent) and the [catalog](#catalog). These are
+described conceptually below to make anti-entropy easier to understand.
+
+<a name="agent"></a>
+#### Agent
+
+Each Consul agent maintains its own set of service and check registrations, as
```

Comment:
If it would help, we can just merge this and then you can edit. Or feel free to push more commits to this branch!